sdk/samples/iot/templates/iothub/test-resources-post.ps1

```diff
@@ -65,12 +65,30 @@ Add-AzIotHubDevice `
 -PrimaryThumbprint $fingerprint `
 -SecondaryThumbprint $fingerprint
 
+if ($LASTEXITCODE -ne 0)
+{
+  Write-Host "Adding cert device failed: LAST_ERROR_CODE=${LAST_ERROR_CODE}"
+  exit $LASTEXITCODE
+}
+
 # Download Baltimore Cert
```

Comment:
Turn this comment into a `Write-Host` prior to running the command, so we get some log info about the steps even when things are running successfully.